### DISCUSSION TITLE
What is the workflow for working on the Website project for DNN?

### DISCUSSION THREAD

**Discussion by jamesehly** [dnnsoftware/Dnn.Platform#4844] [General]
I am confused as to how best work on the website part of DNN.  For example, if I want to change something in the Website/admin directory or modify a portion of the Install.  From the docs it sounds like I should be able to do a simple rebuild of the Website project after making changes, but the website project will not build for me, and from reading other posts, it sounds like the Website project isn't intended to be built.  Is that correct, and if so, what is the correct workflow for making changes to the Website project and then seeing them on the dev site?

**Comment by valadas** [comment]
The website project builds for me, what errors are you getting ?
